pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -123,6 +123,19 @@ type PropagationSpec struct {
 	// If this value is nil, failover is disabled.
 	// +optional
 	Failover *FailoverBehavior `json:"failover,omitempty"`
+
+	// ConflictResolution declares how potential conflict should be handled when
+	// a resource that is being propagated already exists in the target cluster.
+	//
+	// It defaults to "Abort" which means stop propagating to avoid unexpected
+	// overwrites. The "Overwrite" might be useful when migrating legacy cluster
+	// resources to Karmada, in which case conflict is predictable and can be
+	// instructed to Karmada take over the resource by overwriting.
+	//
+	// +kubebuilder:default="Abort"
```

Comment:
the semantics of `Abort` in Git also include undoing the changes made, but we just stop propagating.

Reply:
Yeah I get it. seems we have nothing to `undo` or revert, just stops. That's the meaning of `abort`.

Reply:
> `Abort` means to terminate or cancel something, especially a planned action or process.

From chartGPT 😂